### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by vijaytembugade**
- Added homepage 
- Added Footer
- Added carousels

### CODE REVIEW

File: src/Components/MainCarousel/MainCarousel.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  border: 1px solid var(--text-color);
+  border-radius: 0.6rem;
+  position: absolute;
+  bottom: 5rem;
+  right: 5rem;
+  cursor: pointer;
+}
+.carousel-text:hover {
+  box-shadow: rgba(0, 0, 0, 0.19) 0px 10px 20px, rgba(0, 0, 0, 0.23) 0px 6px 6px;
+  transform: scale(1.1);
```
Comment (t007rushi) [inline src/Components/MainCarousel/MainCarousel.css:22]: with transform just add transitions for smoothness

File: src/Components/SubCarousel/SubCarousel.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        responsive={responsive}
+        pauseOnMouseOver={true}
+        autoSlide={false}
+        infinite={false}
+        showArrows={true}
+        padding="20px"
+      >
+        {writersData.map((photo) => {
+          return (
+            <div className="sub-carousel">
```
Comment (t007rushi) [inline src/Components/SubCarousel/SubCarousel.js]: id?

**Review by t007rushi — APPROVED** [review]
Liked the overall theme #dard-dukh-peeda
Awesome UI 🔥, get time to add dark also
keep it up Thala 👍

File: src/Components/SubCarousel/SubCarousel.js (hunk 2 above)
Comment (MiheerTamkhane) [reply src/Components/SubCarousel/SubCarousel.js]: I think Key!

File: src/App.js [hunk 3]
```diff
@@ -15,15 +15,19 @@ import { ScrollToTop } from "./Components/ScrollToTop/ScrollToTop";
 import Likes from "./Pages/Likes/Likes";
 import WatchLater from "./Pages/WatchLater/WatchLater";
 import History from "./Pages/History/History";
+import Homepage from "./Pages/Homepage/Homepage";
+import Footer from "./Components/Footer/Footer";
```
Comment (MiheerTamkhane) [inline src/App.js:19]: Why are you not using barrel export?

File: src/Components/Footer/Footer.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+      <div className="flex-container-verticle">
+        <div>
+          <strong>
+            Made by <span className="danger-text">Vijay Tembugade</span>
+          </strong>
+        </div>
+        <div className="social-media-icons">
+          <a href="https://github.com/vijaytembugade/" target="_blank">
+            <img src="/assets/social/github.svg" alt="" />
```
Comment (MiheerTamkhane) [inline src/Components/Footer/Footer.js:31]: fill the alt tag for good.

**Review by MiheerTamkhane — APPROVED** [review]
All looks good code is well written as well as structured. 
Also working well and smooth.
### Great work

File: src/Components/MainCarousel/MainCarousel.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            src="/assets/carousel/piyush-mishra.jpg"
+            alt=""
+          />
+        </div>
+
+        <div>
+          <img
+            className="carousel-img"
+            src="/assets/carousel/Rahat-indori.jpg"
+            alt=""
```
Comment (manojasarsa) [inline src/Components/MainCarousel/MainCarousel.js]: You can give the title of images as `alt` attribute

**Review by manojasarsa — APPROVED** [review]
Home page looks good!
GTG!

File: src/App.js (hunk 3 above)
Comment (vijaytembugade) [reply src/App.js:19]: I will refactor

**Pull request merged**